jenkins-master/jobs/scripts/workspace/runtests.py

```diff
@@ -136,6 +136,10 @@ def query_args(self):
 
         return args
 
+class MediaRunner(BaseRunner):
+
+    def __init__(self, *args, **kwargs):
+        BaseRunner.__init__(self, *args, **kwargs)
```

Comment:
Or keep `BaseRunner` and don't create a new class at all given that no additional changes are necessary.